src/config/dynamic-theme-fixes.config

```diff
@@ -12741,6 +12741,21 @@ CSS
 
 ================================
 
+svelte.dev
+
+CSS
+pre.language-svelte {
+    background-color: black;
+}
+.token.keyword {
+    color: #9a94f8;
+}
+span.token.string {
+    color: yellow;
```

Comment:
I don't see anything wrong with the current color:
![image](https://user-images.githubusercontent.com/25481501/141681255-b8713071-8649-42a2-a6f2-7f90256f0ee1.png)